### PR TITLE
MinimizeContext

### PR DESCRIPTION
**Pull request by ue71603**
A flag ist added to the requests to minimize the context information that is returned.

### CODE REVIEW

**Comment by pvgrumbkow** [comment]
What is "absolutely essential content"? It would be much better and clearer to add appropriate ContentFilters which cannot be misunderstood.

**Comment by ue71603** [comment]
You are right. I will create a new pull request that is more in line with concrete ContentFilters.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Changed_Files.zip
```
Comment (skinkie) [inline .gitignore:2]: This should be a separate pull request.

File: OJP_Trips.xsd [hunk 2]
```diff
@@ -168,6 +168,11 @@
 				<xs:annotation>
 					<xs:documentation>Whether the result should include operating day information - as encoded bit string and in natural language.</xs:documentation>
 				</xs:annotation>
+			</xs:element>	
+			<xs:element name="MinimizeResponseContext" type="xs:boolean" default="false" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>hether the ResponseContext should be minimized to absolut essential content. As it is useful in many use cases. The default false means, that the context is sent.</xs:documentation>
```
Comment (skinkie) [inline OJP_Trips.xsd:174]: hether -> whether

**Comment by pvgrumbkow** [comment]
This pull requests has two issues:
1. It's derived from master instead of changes_for_v1.1.
2. It should use specific content filters, e.g. "IncludePlaceContext", "IncludeSituationContext" as optional elements with default "true".
